01-node-tutorial/answers/content/11-fs-async.js

```diff
@@ -0,0 +1,35 @@
+const { readFile, writeFile } = require("fs");
+console.log('begin')
+
+// must add language code or utf8 to prevent a buffer being created
```

Comment:
Can you do a little more research here & revise your comment with what you learn.

In particular, try to answer these questions:

- is utf8 a language code?
- what happens if we don't specify "utf8?" A buffer is _not_ created? Can you find a source for that?